cves/kernel/CVE-2020-12654.yml

```diff
@@ -110,9 +124,9 @@ vcc_instructions: |
   Place any notes you would like to make in the notes field.
 vccs:
 - commit: 5e6e3a92b9a4c9416b17f468fa5c7fa2233b8b4e
-  :note: Discovered automatically by archeogit.
+  :note: Discovered automatically by archeogit. Manually confirmed.
```

Comment:
here the ' : ' before 'note' is not needed I believe. I know the yml file was originally like this but it doesn't match the syntax of the rest of the file. I took mine out and it built and passed all checks.